### PR TITLE
Fix padding issue

### PR DESCRIPTION
**Pull request by brhou**
Fixes the gap issue here:

screen shot 2016-02-24 at 6 49 25 pm

locally, it looks like this, (without the header and without the gap)

screen shot 2016-02-24 at 6 51 14 pm

### CODE REVIEW

File: src/main/html/css/api-explorer.css [hunk 1]
```diff
@@ -1981,6 +1982,7 @@ body {
 
 .swagger-section .swagger-ui-wrap ul#resources li.resource .resource_header h2.resource_name {
     padding: 0 20px;
+    padding-top: 20px;
```
Comment (billkauf) [inline src/main/html/css/api-explorer.css]: Might as well spell out all four, like, `padding: 20px 20px 0 20px;`.

**Comment by billkauf** [comment]
Two things:

1. Note that this also applies to the individual operations as well.
2. There should probably be a top border as well, to separate the tag sections.

**Comment by brhou** [comment]
Didn't notice much affect on each individual endpoint.

Heres the border.

screen shot 2016-02-25 at 11 14 22 am

**Comment by billkauf** [comment]
:+1:

**Pull request merged**